form/src/main/java/org/dhis2/form/data/FormRepositoryImpl.kt

```diff
@@ -522,9 +523,22 @@ class FormRepositoryImpl(
                 }
             }
 
+        fieldsWithOptionEffects.forEach { field ->
```

Comment:
do you think that this can be covered by a test in `FormRepositoryImplTest`?

Reply:
yes!